Resources/public/js/views/fields/ez-maplocation-editview.js

```diff
@@ -63,9 +63,10 @@ YUI.add('ez-maplocation-editview', function (Y) {
          * @method initializer
          */
         initializer: function () {
-            var mapLoader = this.get('mapAPILoader');
+            var mapLoader = this.get('mapAPILoader'),
+                config = this.get('config');
 
-            mapLoader.load();
```

Comment:
to avoid the jshint error, you can write:
```js
/*jshint camelcase: false */
mapLoader.load(config.apiKeys.google_maps || '');
/*jshint camelcase: true */
```